Automattic/wp-calypso
# Add missing uuid dependency

### Problem

`@yarnpkg/doctor` found an undeclared dependency:
```
➤ YN0000: ┌ /Users/sergio/src/automattic/wp-calypso/packages/calypso-analytics/package.json
➤ YN0000: │ User scripts prefixed with "pre" or "post" (like "prepublish") will not be called in sequence anymore; prefer calling prologues and epilogues explicitly
➤ YN0000: │ User scripts prefixed with "pre" or "post" (like "prepare") will not be called in sequence anymore; prefer calling prologues and epilogues explicitly
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/packages/calypso-analytics/src/train-tracks.ts:4:1: Undeclared dependency on uuid
➤ YN0000: └ Completed in 0.69s
```

### Changes

This PR adds that dependency to `./packages/calypso-analytics/package.json`. The version range has been copied from `./package.json` to make sure nothing changes.

### Testing instructions

Verify there are no more missing packages:
* Run `npx @yarnpkg/doctor` inside `./packages/calypso-analytics`. All warnings about missing dependencies should be gone now.
